account_statement_completion_compassion/model/statement.py

```diff
@@ -259,6 +268,26 @@ def _generate_invoice(self, cr, uid, st_line, partner, context=None):
                     wf_service.trg_validate(
                         uid, 'account.invoice', invoice_id, 'invoice_open',
                         cr)
+                elif product.name == GIFT_TYPES[0]:
+                    # Set date of invoice two months before child's birthdate
+                    child_birthdate = res.get('child_birthdate')
+                    if child_birthdate:
+                        inv_date = datetime.strptime(st_line['date'], DF)
+                        birthdate = datetime.strptime(child_birthdate, DF)
+                        new_date = inv_date
```

Comment:
Ok, so if the code fits the needs, please update the comment. A bad comment is worse than no comment at all...